DJHelper/DJHelper/Model/Event Model/Event Controller/EventController.swift

```diff
@@ -26,6 +27,83 @@ class EventController {
         self.dataLoader = dataLoader
     }
 
+    // MARK: - UPDATE EVENT
+
+    func updateEvent(event: Event,
+                     eventName: String,
+                     eventDate: Date,
+                     description: String,
+                     startTime: Date,
+                     endTime: Date,
+                     type: String,
+                     notes: String) -> Event {
+            event.name = eventName
+            event.eventDate = eventDate
+            event.eventDescription = description
+            event.startTime = startTime
+            event.endTime = endTime
+            event.eventType = type
+            event.notes = notes
+
+        //and save to core data
+        do {
+            try CoreDataStack.shared.save()
+        } catch {
+            print("""
+                Error on line: \(#line) in function: \(#function)\n
+                Readable error: \(error.localizedDescription)\n Technical error: \(error)
+                """)
+        }
+        return event
+    }
+
+    func saveUpdateEvent(_ event: Event, forHost hostID: Host, completion: @escaping (Result<(),EventErrors>) -> Void) {
+        guard let eventRep = event.eventAuthorizationRep else {
+            print("Error on line: \(#line) in function: \(#function)\n")
+            return
+        }
+
+        let authURL = baseURL.appendingPathComponent("auth")
+        let eventURL = authURL.appendingPathComponent("event")
+        let finalURL = eventURL.appendingPathComponent("\(hostID.identifier)")
```

Comment:
Is this URL using the host identifier or the event identifier?  I have not tried it out, but it seems like the event identifier might make more sense than the host identifier.